spec/requests/automations_spec.rb

```diff
@@ -0,0 +1,97 @@
+# frozen_string_literal: true
+
+require_relative '../discourse_automation_helper'
+
+describe DiscourseAutomation::AutomationsController do
+  describe '#trigger' do
+    fab!(:automation) { Fabricate(:automation, script: 'nothing_about_us') }
```

Comment:
I think maybe the tests should be using an actual script like `send_pm`, because as I discovered `send_pm` doesn't work for this new triggerable

Reply:
It does work but yes it needs usernames as context. I thought of it and I think one good solution would be to have trigger/script combo doc keys.

So if api_call + send_pms => I show an alert-info saying your api call needs to have this field. I think hat would be good?

Reply:
Hmm I tried to send usernames and it didn't work, maybe I just had the structure of the POST params wrong. I think 100% adding docs for api_call + send_pms or whatever else you should show an info box with what the payload of the API call needs to be; I was just stabbing in the dark